_capabilities/fault_lines_UCLA

```diff
@@ -0,0 +1,53 @@
+---
+title: "Fault-Line Image Explanations"
+excerpt: "Algorithm for generating conceptual and counterfactual explanations for an image classification model."
+tags:
+  - Analytics
+  - Computer Vision
+  - Saliency
+  - Explanation Framework
+   
+submission_details:
+  resources: 
+    papers:
+      - https://ojs.aaai.org/index.php/AAAI/article/view/5643
+    software:
+      - https://github.com/arjunakula/faultline_explainer
+      - https://github.com/arjunakula/CoCoX
+   
+  # Optional information describing artifact
+  version: 1.0
+  size: 100MB
+  license: NA
+   
+  authors:
+    - Arjun Akula
+    - Song-Chun Zhu
+  organizations:
+    - UCLA
+  point_of_contact:
+    name: Arjun Akula
+    email: aakula@ucla.edu
+---
+   
+## Overview
+We propose this new explanation framework to identify the minimal semantic-level features (e.g. stripes on zebra, pointed ears on dog) referred to as the explainable concept, that can be added/deleted to alter the classification category.
+Please see a demo of this work using the following links: 
+
+1) https://github.com/arjunakula/arjunakula.github.io/blob/master/AAAI2020_poster%20(1).pdf
+
+2) https://www.dropbox.com/s/kde88lgv78v8dyj/Arjun_UCLA_Analytics_v2.pptx?dl=0
```

Comment:
Please make this into a Markdown list with direct links. For example:

```
1. <https://github.com/arjunakula/arjunakula.github.io/blob/master/AAAI2020_poster%20(1).pdf>
2. <https://www.dropbox.com/s/kde88lgv78v8dyj/Arjun_UCLA_Analytics_v2.pptx?dl=0>
```

If you prefer to provide a short name for your links, you can try this instead:

```
1. [Poster](https://github.com/arjunakula/arjunakula.github.io/blob/master/AAAI2020_poster%20(1).pdf)
2. [Presentation](https://www.dropbox.com/s/kde88lgv78v8dyj/Arjun_UCLA_Analytics_v2.pptx?dl=0)
```

You should be able to use the "Preview" tab to render the Markdown.

Reply:
done